axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -883,7 +883,8 @@ InvoiceSet.length > 0))"
     </action-record>
 
 	<action-record name="action-invoice-record-specific-note-on-invoice" model="com.axelor.apps.account.db.Invoice">
-		<field name="note" expr="eval: companyBankDetails?.specificNoteOnInvoice"/>
+		<field name="note" expr="eval: note + companyBankDetails?.specificNoteOnInvoice" if="note"/>
+		<field name="note" expr="eval: companyBankDetails?.specificNoteOnInvoice" if="!note"/>
```

Comment:
can be done in one eval with (note != null ? note + ' ' :  '') +  companyBankDetails?.specificNoteOnInvoice 
Space should be there between old note and new.